src/components/Reviews.jsx

```diff
@@ -0,0 +1,35 @@
+import { getReviews } from "../api"
+import { useEffect } from "react"
+import { useState } from "react"
+
+const Reviews = () => {
+
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        getReviews()
+        .then((reviews) => {
```

Comment:
For readability, I would change this parameter to something other than reviews as you already have a state called reviews on line 7. This represents a different set of data. Maybe newReviews? reviewsFromApi?